docs/references/teams/update-team-name.md

```diff
@@ -0,0 +1 @@
+Update the team's name by its unique ID.
```

Comment:
We should mention that only members with the owner role can update the team name

Reply:
Actually I don't think its protected by the `owner` role.

Reply:
@TorstenDittmann please check, sounds weird if it doesn't.

Reply:
no, works as intended. I somehow mixed things up 👍🏻